src/concrete/ml/sklearn/base.py

```diff
@@ -1280,8 +1280,41 @@ def __init__(self, n_bits: int):
         #: The model's inference function. Is None if the model is not fitted.
         self._tree_inference: Optional[Callable] = None
 
+        #: Boolean that indicates whether the rounding feature is enabled or not.
+        self._use_rounding: bool = True
+
         BaseEstimator.__init__(self)
 
+    @property
+    def use_rounding(self) -> bool:
+        """The rounding property.
+
+        Returns:
+            bool: Whether to enable or disable rounding
+        """
+        return self._use_rounding  # pragma: no cover
+
+    @use_rounding.setter
+    def use_rounding(self, value: bool) -> None:
```

Comment:
please make this a private property. this should not be visible to the user nor maintained as a public API